### ISSUE TITLE
2.1.26 Visual Problems

### ISSUE THREAD

**Issue by keithcdodd** [my-Anki/Beautify-Anki#12]
Really amazing work. So far, not compatible with 2.1.26 (debian Anki) as desired. 

Some issues noted thus far:
(1) Heatmap not wrapping or appearing correctly [probably a heatmap issue since that is not working well with 2.1.26 as it is either)
(2) Visual with white on white (ameloriated by turning off night mode, although the visuals when not on night mode are not as desired either)
(3) When clicking on a deck to study, there is some added text
(4) Just generally not fully functioning for 2.1.26 I think. I can look into more details if desired, I just turned it off for now.

Really cool, thanks so much for doing this.

**Comment by ShoroukAziz** [comment]
Thanks for reporting. Could you post screenshots, please?

**Comment by keithcdodd** [comment]
Sure! I attached some with and without nightmode. For the "added text" I am referring to the "exit_to_app" next to "Study Now".

Again, thank you for this. Overall, this is very cool. 

![Nightmode_DeckSelected](https://user-images.githubusercontent.com/61259563/82512122-63683100-9acc-11ea-847a-5d555eae7560.png)
![NightMode_DoingCards](https://user-images.githubusercontent.com/61259563/82512123-6400c780-9acc-11ea-8a50-b7008c9b8f14.png)
![Nightmode_Overall](https://user-images.githubusercontent.com/61259563/82512125-64995e00-9acc-11ea-8b34-713f86e4c06e.png)
![NoNightmode_DeckSelected](https://user-images.githubusercontent.com/61259563/82512128-65ca8b00-9acc-11ea-8e09-2dddefac1702.png)
![NoNightmode_Overall](https://user-images.githubusercontent.com/61259563/82512129-65ca8b00-9acc-11ea-83c8-70e879538113.png)

**Comment by ShoroukAziz** [comment]
Hi again,
Thanks a lot, for following up I appreciate it.
I had tested the addon on version 2.1.26 myself on another computer and now I installed it myself
but on both devices, I had no problems. So, I'm trying to reproduce the errors you're having so I could start solving it. I'm gonna need a few things, So, please, help me with them.

First, could you please confirm the following :
- [ ] You have the  [official Review Heatmap by Gluanimate. ] (https://github.com/glutanimate/review-heatmap/releases) and not any other version
- [ ] You have disabled [More Decks Stats and Time Left addon](https://ankiweb.net/shared/info/1556734708) and any addon that does a similar job.
- [ ] You have disabled [More Overview Stats 2.1 addon](https://ankiweb.net/shared/info/738807903) and any addon that does a similar job.

I also need to know Which version of night mode are you using? (the one I found on Ankiweb only supports up to Anki 2.1.19)

Finally, It would be super helpful if you could post a screenshot of your Anki home screen and deck page with Beautify Anki disabled. some people are having the same problem of different colors appearing (I noticed that the gear icons in your screenshots are blue while they're supposed to be orange! also, the numbers of reviews in the deck tables are supposed to be white but they have a greenish color so that's crazy!)

**Comment by keithcdodd** [comment]
Hello,

Thank you! I have confirmed I have the official Review Heatmap. I did not have More Decks Stats and Time Left, BUT, I did have More Overview Stats 2.1 (sorry for missing that!). Also, I am not using an addon for nightmode; my apologies for the confusion. Rather I am using it from within Anki (Tools -> Preferences -> Night Mode). 

After disabling More Overview Stats 21, this is what I have:
![Screenshot from 2020-05-21 09-24-54](https://user-images.githubusercontent.com/61259563/82575004-22f8c980-9b45-11ea-8fd1-94de2bb5d743.png)
![Screenshot from 2020-05-21 09-25-16](https://user-images.githubusercontent.com/61259563/82575009-2429f680-9b45-11ea-9ec7-80e579940ad6.png)
![Screenshot from 2020-05-21 09-25-43](https://user-images.githubusercontent.com/61259563/82575011-24c28d00-9b45-11ea-9f10-c43a6f6bb3d7.png)

Here are screenshots with your addon disabled too:
![Screenshot from 2020-05-21 09-27-18](https://user-images.githubusercontent.com/61259563/82575153-520f3b00-9b45-11ea-9218-f180c7c2a239.png)
![Screenshot from 2020-05-21 09-27-26](https://user-images.githubusercontent.com/61259563/82575158-52a7d180-9b45-11ea-94fe-e6528ee4776a.png)
![Screenshot from 2020-05-21 09-27-57](https://user-images.githubusercontent.com/61259563/82575211-62271a80-9b45-11ea-94ae-8f9fec4b799e.png)

Finally, here is my current list of addons:
![Screenshot from 2020-05-21 09-29-05](https://user-images.githubusercontent.com/61259563/82575351-9b5f8a80-9b45-11ea-8806-45258ebc280e.png)
![Screenshot from 2020-05-21 09-29-16](https://user-images.githubusercontent.com/61259563/82575353-9bf82100-9b45-11ea-9e81-84bac5b1ee25.png)

**Comment by keithcdodd** [comment]
Oh, one more thing. For the addon "Automatic Scoring 2" what it does is depending on how long it takes me to flip the card, it will highlight and autoselect an answer (easy, good, hard, again) so that when I hit spacebar, it selects that answer. 

So, when I flip a card, it will highlight (by putting a round orange box around) the answer that it will hit when I hit spacebar. (then I don't have to think about difficulty and such). With your addon, this sort of highlight feature is gone, so I do not know if the addon is working and/or which answer is being selected by my spacebar. It would be really cool if the highlight aspect from this type of addon would still function.

**Comment by ShoroukAziz** [comment]
Hi, Sorry for the late response. I got real busy
Any way, This was very helpful. I'll let you know if I need anything else. Hopefully in 10 days most of these issues will be fixed.
Also, I'll try to make the addon compatible with "Automatic Scoring 2".

**Comment by keithcdodd** [comment]
Hello, thanks for the new release! Most things are better! Oddly I am getting this though when I open a deck:
![image](https://user-images.githubusercontent.com/61259563/83373838-09e2ea80-a387-11ea-9a52-a2bda89305a6.png)

Here are my addons... I tried to disable ones that I thought would conflict...
![image](https://user-images.githubusercontent.com/61259563/83373890-23843200-a387-11ea-91cd-ec1758be4bc0.png)

I am pretty confident these are the same addons I had with the previous update where this page looked close to normal (mines the exit to app writing which is still showing for me).

Thanks!

**Comment by keithcdodd** [comment]
Hello, I am sorry but I also just noticed that since adding on the new version of this, my addon "Speed focus mode auto-alert auto-reveal auto-fail" is no longer working -- e.g. I have it set to auto-reveal at 9 seconds (which has worked fine in the past) but it no longer auto-reveals at all. I can check if it is a problem with that addon, but since I haven't changed that addon, and this only started happening after I updated this one, I am guessing it is somehow conflicting.
